packages/TokenRunner/src/Analyzer/FixerAnalyzer/IndentDetector.php

```diff
@@ -19,11 +19,15 @@ public function __construct(WhitespacesFixerConfig $whitespacesFixerConfig)
 
     public function detectOnPosition(Tokens $tokens, int $startIndex): int
     {
+        $indent = $this->whitespacesFixerConfig->getIndent();
+
         for ($i = $startIndex; $i > 0; --$i) {
             $token = $tokens[$i];
 
-            if ($token->isWhitespace() && $token->getContent() !== ' ') {
-                return substr_count($token->getContent(), $this->whitespacesFixerConfig->getIndent());
+            $lastNewlinePos = strrpos($token->getContent(), "\n");
```

Comment:
Maybe PHP_EOL or newline from whitespacesFixerConfig would be more precise

Reply:
Not really. You are actually doing a mistake by using PHP_EOL in your fixers. Value of this constant is runtime specific and will mean running the fixer on Windows computer won't fix code code with Unix style line endings. `\n` is actually more portable here.

Reply:
Agreed. The point it's user configured.

Reply:
I've tried to use `$indent` it broke. So you're right, it need to be "\n", probably due to in string position.

Reply:
I think you had in mind to use getlinelength, not $indent

Reply:
Nono, I meant $indent, but it doesn't work as I expected.